gdk/aws_clients/Greengrassv2Client.py

```diff
@@ -1,65 +1,49 @@
 import logging
+import boto3
 
 
 class Greengrassv2Client:
     """
     Greengrasv2 client utils wrapper
     """
 
-    def __init__(self, project_configuration, service_clients):
-        self.project_config = project_configuration
-        self.greengrass_client = service_clients["greengrass_client"]
+    def __init__(self, _region):
+        self.client = boto3.client("greengrassv2", region_name=_region)
 
-    def get_highest_component_version_(self) -> str:
```

Comment:
maybe rename this since it is doing something different than before. Name `get_highest_cloud_component_version`?